main/guides/subquery-indexing.md

```diff
@@ -0,0 +1,32 @@
+# SubQuery Indexing
+
+This document explains how to index Agoric blockchain data with SubQuery's open source data indexer.
+
+## Overview
+
+SubQuery is a leading blockchain data indexer that provides developers with fast, flexible, universal, open source and decentralised APIs for web3 projects. SubQuery SDK allows developers to get rich indexed data and build intuitive and immersive decentralised applications in a faster and more efficient way. SubQuery supports 100+ ecosystems including Agoric Cosmos, Ethereum, Polygon, Polkadot, Algorand, NEAR, and Avalanche.
+
+Features include multiple RPC endpoint configurations, multi-worker capabilities and a configurable caching architecture. To find out more, visit our [documentation](https://academy.subquery.network/).
```

Comment:
@sufyaankhan is Agoric willing to back up these claims by putting them in our official documentation, or should they be clarified as being made by the SubQuery team (or be made less strong)?

I would be comfortable with:
```suggestion
As described by the SubQuery team:
> SubQuery is a leading blockchain data indexer that provides developers with fast, flexible, universal, open source and decentralised APIs for web3 projects. SubQuery SDK allows developers to get rich indexed data and build intuitive and immersive decentralised applications in a faster and more efficient way. SubQuery supports 100+ ecosystems including Agoric Cosmos, Ethereum, Polygon, Polkadot, Algorand, NEAR, and Avalanche.
>
> Features include multiple RPC endpoint configurations, multi-worker capabilities and a configurable caching architecture. To find out more, visit our [documentation](https://academy.subquery.network/).
```

Reply:
+1 - @michaelfig I am also comfortable with the language you have suggested.

Reply:
Thanks @sufyaankhan added the suggestion.